tjenestespesifikasjoner/pdl-api/src/main/resources/pdl/queries/hentPersondata.graphql

```diff
@@ -249,5 +249,23 @@ query($ident: ID!){
                 ...utenlandskadresse
             }
         }
+        oppholdsadresse {
+            gyldigFraOgMed
```

Comment:
Burde vi lagt til gyldigheten for andre adressetyper også? E.g bosted og kontaktadresse har vel begge samme konseptet.